doc/user/content/third-party/supported-tools.md

```diff
@@ -1,33 +1,216 @@
+| Java | 🟢 Production | The popular [PostgreSQL JDBC driver](https://jdbc.postgresql.org/) can be used to interact with Materialize as if it were a PostgreSQL DB. |  |
+| Golang | 🟢 Production | TODO: Confirm which Go drivers have been tested. |  |
+| PHP | 🟢 Production | The standard PHP [PostgreSQL Extension](https://www.php.net/manual/en/ref.pgsql.php) can be used to interact with Materialize as if it were a PostgreSQL DB. |  |
+
+## Frameworks and ORMs
+
+Frameworks and ORMs tend to make more advanced queries to PostgreSQL behind the scenes, using configuration settings and system table endpoints that Materialize hasn't yet implemented. As Materialize [expands `pg_catalog` support](https://github.com/MaterializeInc/materialize/issues/2157), support for frameworks and ORMs will improve.
+
+| Service | Materialize Support | Notes |  |
+| --- | --- | --- | --- |
+| Ruby on Rails | 🔴 Researching | The Rails Active-Record ORM executes many PostgreSQL configuration queries and queries to `pg_catalog` endpoints that are not yet implemented in Materialize. | [](#notify) |
+| Prisma | 🔴 Researching | Prisma executes configuration queries and queries to `pg_catalog` endpoints that are not yet implemented in Materialize. | [](#notify) |
+
+_Is there another framework or ORM you'd like to use with Materialize? [Open a GitHub Issue here](https://github.com/MaterializeInc/materialize/issues/new?assignees=&labels=A-integration&template=02-feature.yml)._
+
+## Data Tools
+
+Many tools in the modern data stack can connect to Materialize via PostgreSQL, but like ORMs, these tools sometimes make advanced configuration and system table queries to endpoints that Materialize hasn't yet implemented.
+
+### Business Intelligence (BI) Tools
+
+| Service | Materialize Support | Notes |  |
+| --- | --- | --- | --- |
+| Metabase | 🟢 Beta | The Metabase PostgreSQL connector can be used to [connect Metabase to Materialize Core](/third-party/metabase/). | [More Info](/third-party/metabase/) [](#notify) |
+| Superset | 🟢 Beta | TBD @andrioni input. | [](#notify) |
+| Looker | 🟡 Alpha | TBD @andrioni input. | [](#notify) |
+| Google Data Studio | 🟡 Alpha | Google Data Studio can connect to Materialize Core and Cloud using the PostgreSQL connector. Data is cached hourly but can be manually refreshed. | [](#notify) |
+| Tableau | 🔴 Researching | | [](#notify) |
+| Microsoft Power BI | 🔴 Researching | | [](#notify) |
+| Preset | 🔴 Researching | | [](#notify) |
+| Mode Analytics | 🔴 Researching | | [](#notify) |
+| Holistics BI | 🔴 Researching | | [](#notify) |
+
+### Other Data Applications and Tools
+
+| Service | Materialize Support | Notes |  |
+| --- | --- | --- | --- |
+| Hex | 🟡 Alpha | Users of Hex can connect to Materialize Core instances via the Hex PostgreSQL connector. Hex automatically introspects Materialized Views and Tables. *(Cloud connectivity is blocked by user/password auth.)* | [](#notify) |
```

Comment:
@JLDLaughlin Can you suggest a status and note for Hex here

Reply:
Yes, sorry for my late response!

Status: alpha seems fine! Let's bump to beta once we've worked the partnership launch through with them. (And the known bugs.)
Note: it looks perfect to me! I'll open a PR when the user/password auth is fixed and when we get a custom connector logo.